src/Domain/SalesforceWriteProxy.php

```diff
@@ -38,6 +38,22 @@ public function getSalesforceLastPush(): ?DateTime
         return $this->salesforceLastPush;
     }
 
+    /**
+     * @return DateTime
+     */
+    public function getCreatedAt(): ?DateTime
```

Comment:
Shouldn't be in base class as that's not what uses the trait which defines the field.